template-parts/entry-header.php

```diff
@@ -15,10 +15,14 @@
 
 		<?php
 
-		if ( has_category() ) {
+		// Allow child themes and plugins to filter the display of the categories in the entry header.
+		$show_categories = apply_filters( 'twentytwenty_show_categories_in_entry_header', true );
```

Comment:
Could we use the special return functions here instead of static bools? https://developer.wordpress.org/reference/functions/__return_true/